### DISCUSSION TITLE
Condition to have a field Hidden on List

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#14386] [Feature Requests]
Currently we may set the condition `Hidden on Detail`. 
They are situations where we don't want a user to be able to add the corresponding column in the list.
It would be nice to be able to set `Hidden on List`.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
